libpod/container_internal_linux.go

```diff
@@ -316,6 +319,19 @@ func (c *Container) generateSpec(ctx context.Context) (*spec.Spec, error) {
 		}
 	}
 
+	// Add overlay volumes
+	for _, overlayVol := range c.config.OverlayVolumes {
+		contentDir, err := overlay.TempDir(c.config.StaticDir, c.RootUID(), c.RootGID())
```

Comment:
Should this be the root UID/GID or the UID/GID of the user running in the container?